isb_web/api_types.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+from isb_web.isb_enums import ISBAuthority, ISBReturnField
+
+"""
+Module for various API parameter and return types used throughout iSamples API
+"""
```

Comment:
Moved this out to its own file as `main.py` has grown a bit long in the tooth and this seemed like a clean way of organizing things while trimming down a bit of that cruft.